engine/src/main/java/org/terasology/engine/rendering/dag/RenderGraph.java

```diff
@@ -339,8 +342,8 @@ private void connectBufferPair(Node toNode, int inputConnectionId, DependencyCon
 
         } else { // if adding new input failed, it already existed - check for connections
             //TODO update
-            logger.info("{}.connectFbo({}, {}): Connection already existed. Testing for its connections..",
-                    toNode.getUri(), inputConnectionId, fromConnection.getName());
+            logger.atInfo().addArgument(() -> toNode.getUri()).addArgument(inputConnectionId).addArgument(() -> fromConnection.getName()).
+                    log("{}.connectFbo({}, {}): Connection already existed. Testing for its connections..");
```

Comment:
simple getter methods
```suggestion
            logger.info("{}.connectFbo({}, {}): Connection already existed. Testing for its connections..",
                    toNode.getUri(), inputConnectionId, fromConnection.getName()); //NOPMD
```